File: Deafios em C#/2936 - Quanta Mandioca.cs
using System;

namespace _2936___Quanta_Mandioca
{
    class Program
    {
        static void Main() {

            int chico = 300 * Int32.Parse(Console.ReadLine());
            int bento = 1500 * Int32.Parse(Console.ReadLine());
            int bernardo = 600 * Int32.Parse(Console.ReadLine());
            int marina = 1000 * Int32.Parse(Console.ReadLine());
            int iara = 150 * Int32.Parse(Console.ReadLine());
            int marlene = 225;
            int total = chico + bento + bernardo + marina + iara + marlene;

            Console.WriteLine(total);
        }
    }
}

File: Desafios em JavaScript/2936 - Quanta Mandioca.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var chico = 300 * parseInt(lines.shift());
var bento = 1500 * parseInt(lines.shift());
var bernardo = 600 * parseInt(lines.shift());
var marina = 1000 * parseInt(lines.shift());
var iara = 150 * parseInt(lines.shift());
var marlene = 225;
var total = chico + bento + bernardo + marina + iara + marlene;

console.log(`${total}`);